### DISCUSSION TITLE
selling my LMN 3

### DISCUSSION THREAD

**Discussion by fichone** [FundamentalFrequency/.github#86] [General]
hey guys,

decided to sell my LMN 3. it includes everything - raspberry pi 4, teensy 4.1, touch screen, SD card with the software, etc. Fully working and ready to be used.

link to YouTube video how I built it:

https://youtube.com/watch?v=B7E4KHge6iY&feature=share8

If anyone interested, please ping me. Since I am based in Bulgaria, prefer shipping EU only. New owner will receive another unused PCB for free.

**Comment by yratof** [comment]
Hey, How much are you thinking? Not able to DM on github @fichone

**Reply by fichone** [reply]
ferferiev@gmail.com

**Comment by fichone** [comment]
You can send me email here: ***@***.***

**Comment by mardab** [comment]
Is this still available?

**Comment by fichone** [comment]
Yes, it is available.

**Comment by yratof** [comment]
I was very tempted, but the developer has abandoned the project, so it scares me a little to considerOn 1 Sep 2023, at 21:17, fichone ***@***.***> wrote:﻿
Yes, it is available.



On Fri, Sep 1, 2023, 10:07 PM Marcin Dąbrowski ***@***.***>

wrote:

**Reply by fichone** [reply]
Ah, you were the one asking me about the price, but never wrote back at least to say what was your final decision! Now I am not sure why you are sharing your opinion in my post (someone asked you to do so?!?) Also who told you that the project is abandoned?

**Comment by yratof** [comment]
Oh sorry I’m replying via email, didn’t realise it was a thread on GitHub
still - search for this on Reddit, you can see that the guy who made it
hopes the community picks it up and develops it further, but he hasn’t
touched it since he released it. I’ll find the link.

However, if the community is there, I don’t see what it can’t have MPC
style software in there that’s bug free
